site/ja/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contribution Guideline
+
+## Communities
```

Comment:
I think this Community section should go in the ja/README.md file. This will make it easier for users browsing GitHub to see.

Reply:
I removed this subsection and updated README.md.
Please check it.